### ISSUE TITLE
DRY : Don't Repeat Yourself

### ISSUE THREAD

**Issue by mairouche** [maguedon/synapse#1]
Bonjour Mathilde,

Tu as dû remarquer en implémentant la liste des 5 races les plus jouées que tu avais comme une odeur de code : tes méthodes de StatistiquesServiceImpl ainsi que tes classes XXXMostPlayedVO sont quasi identiques.

Ca sent le refactoring :D

**Comment by maguedon** [comment]
Oui, on s'en est vite rendu compte. Il faudra qu'on s'occupe de refactorer.
